spec/ruby_llm/chat_spec.rb

```diff
@@ -28,7 +27,20 @@
       it "#{model} can handle multi-turn conversations" do # rubocop:disable RSpec/MultipleExpectations
         chat = RubyLLM.chat(model: model)
 
-        first = chat.ask("Who was Ruby's creator?")
+        first =
+          if model =~ /llama/
+            # HACK: provisional code just to exemplify a problem with the original question.
+            # llama3.1:8b takes "Ruby's creator" to refer to some fictional character,
+            # but apparently "creator of Ruby" sounds less about a person and gives it a little helpful push.
+            #
+            # Ideally the question could be changed and other cassettes re-recorded;
+            # otherwise this can probably be solved with a bigger model but it will make the repeating Ollama tests
+            # more computationally expensive for everyone.
+            chat.ask('Who was the creator of Ruby?')
```

Comment:
@crmne I can reliably reproduce llama3.1:8b thinking "who was Ruby's creator" is about a fictional character, whereas it gets the above form mostly correct (takes a few tries; L3 8B is nowhere near a frontier model).

This is provisional; it would be great if you could change the question and refresh cassettes. Failing that, I can look for a stronger model (but it will be heavier).